app/routes/my-projects.js

```diff
@@ -1,4 +1,7 @@
 import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
-export default class MyProjectsRoute extends Route {
+const AuthenticatedRoute = Route.extend(AuthenticatedRouteMixin);
```

Comment:
will we also need to add this to app/routes/show-project.js?

Reply:
@godfreyyeung nope — you don't need to be authenticated to look at the whole route. You only need to be authenticated to see the buttons for hearings/rec. Right?